main.go

```diff
@@ -175,3 +212,27 @@ func restConfigForAPIExport(ctx context.Context, cfg *rest.Config, apiExportName
 
 	return cfg, nil
 }
+
+func apisKcpGroupPresent(restConfig *rest.Config) bool {
```

Comment:
(no action required) I'd probably call this `kcpAPIsGroupPresent`

Reply:
renamed 77c8d19